internal.go

```diff
@@ -0,0 +1,65 @@
+package config
+
+type Internal struct {
+	Bitswap InternalBitswap
+}
+
+const (
+	DefaultBitswapEngineBlockstoreWorkerCount = 2000
+	DefaultBitswapTaskWorkerCount             = 500
+	DefaultBitswapEngineTaskWorkerCount       = 500
+	DefaultBitswapMaxOutstandingBytesPerPeer  = 1 << 21
+)
```

Comment:
AFAICT we don't have other defaults defined in this package. Let's move this into go-ipfs where we don't have to do a version bump to make a change (and it's more similar to how we handle other defaults)

Reply:
For reference the numbers defined in go-bitswap and that we'd probably want to use by default are:
```
	defaulEngineBlockstoreWorkerCount = 128

	// how many worker threads to start for decision engine task worker
	defaultEngineTaskWorkerCount = 8
	// the total number of simultaneous threads sending outgoing messages
	defaultTaskWorkerCount = 8
	// the total amount of bytes that a peer should have outstanding, it is utilized by the decision engine
	defaultEngineMaxOutstandingBytesPerPeer = 1 << 20
```

We could also defer to using the numbers in go-bitswap, but we can have that discussion in the go-ipfs PR.